lib/circuitry/config/file_loader.rb

```diff
@@ -0,0 +1,24 @@
+require 'yaml'
+require 'erb'
+require 'fileutils'
+
+module Circuitry
+  module Config
+    module FileLoader
+      def self.load(cfile, environment = 'development')
+        return nil unless File.exist?(cfile)
+
+        opts = {}
+        opts = YAML.load(ERB.new(IO.read(cfile)).result) || opts
+        opts = opts.merge(opts.delete(environment) || {})
```

Comment:
Am I interpreting this correctly that we're allowing either top-level YAML settings *or* environment-specific YAML settings?